tests/test_gcp_text_moderation_input_rail.py

```diff
@@ -0,0 +1,167 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+import pytest
+from google.cloud import language_v2
```

Comment:
Try to follow the same pattern as in https://github.com/NVIDIA/NeMo-Guardrails/blob/develop/nemoguardrails/library/sensitive_data_detection/actions.py so that this doesn't fail when the Google libraries are not installed and the rail is not used.